tutor/src/models/tour/context.js

```diff
@@ -88,14 +88,17 @@ class TourContext extends BaseModel {
   }
 
   @computed get activeRegion() {
-    if (!this.tour){ return null; }
+    if (!this.tour) { return null; }
     return this.regions.find(region => region.tour_ids.find(tid => tid === this.tour.id));
   }
 
+  // terms agreements are allowed to interrupt tours
+  @computed get ready() {
+    return (isEmpty(this.courses) || !User.terms_signatures_needed) && !!this.tour;
+  }
+
```

Comment:
hah! so it's not my fault that this isn't `isReady` 😁   It should also be either wrapped in a `Boolean` or a `!!` - I personally like `Boolean` but I'm in the minority on that, you'll often see the `!!`.

So either:
```
  return Boolean( (isEmpty(this.courses) || !User.terms_signatures_needed) && this.tour );
```
or:
```
  return !!( (isEmpty(this.courses) || !User.terms_signatures_needed) && this.tour );
```